package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "2.4.1",
+  "version": "2.4.2",
```

This change constitutes a minor version
```suggestion
  "version": "2.5.0",
```